src/patternfly/components/Table/examples/table-width-example.hbs

```diff
@@ -49,7 +49,7 @@
         <input type="checkbox" name="checkrow2" aria-labelledby="width-node2">
       {{/table-td}}
       {{#> table-th table-th--data-label="Repository Name"}}
-        <p id="{{table--id}}-node2">Node 2</p>
+        <p id="width-node2">Node 2</p>
```

Comment:
Why not do the `table--id` as the other examples have?

Reply:
good point will update it while im at it